### ISSUE TITLE
Write API Method returns 400 bad timestamp on multiple inserts

### ISSUE THREAD

**Issue by jmorantebriones** [influxdb/influxdb#4551]
I was using influxdb 0.9.0 and it was working fine, but I have upgraded to 0.9.4.2 and it returns "bad timestamp" error when sending multiple inserts (that don't include timestamp). I am running Influx in a Linux server (Linux 4.1.7-15.23.amzn1.x86_64 #1 SMP Mon Sep 14 23:20:33 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux) and I installed the software using the RPM file.

If I try to send one by one it works fine, but when I send multiple lines it fails sometimes (not always). It shouldn't be an issue related with "\n" because it works fine with 0.9.0. I have double checked it, I have installed a new server with 0.9.0 and confirmed that with that version it works fine.

I send the following content in the POST call:

```
metric,agentname="JMB_test2210",name="rtPerUrl",parent="Welcome\ to\ Tomcat",path="tags" value=1.0,units="ms"
metric,agentname="JMB_test2210",name="hitsPerUrl",parent="Welcome\ to\ Tomcat",path="tags" value=1.0,units="hits"
metric,agentname="JMB_test2210",name="cpuPerUrl",parent="Welcome\ to\ Tomcat",path="tags" value=0.0,units="%"
```

And I am calling the following URL: http://127.0.0.1:8086/write?db=tests_db

For the tests of the API I am using DHC Chrome application: 

![image](https://cloud.githubusercontent.com/assets/11889002/10698646/5d5395d8-79b2-11e5-9c4a-a8026b225566.png)

![image](https://cloud.githubusercontent.com/assets/11889002/10698698/ae469ed6-79b2-11e5-9d49-2d199675b8af.png)

**Comment by beckettsean** [comment]
Your [line protocol syntax](https://influxdb.com/docs/v0.9/write_protocols/write_syntax.html) is incorrect, unless you want the double-quotes to actually be stored as part of the value. 

Try submitting the points with the following syntax instead:

```
metric,agentname=JMB_test2210,name=rtPerUrl,parent=Welcome\ to\ Tomcat,path=tags value=1.0,units="ms"
metric,agentname=JMB_test2210,name=hitsPerUrl,parent=Welcome\ to\ Tomcat,path=tags value=1.0,units="hits"
metric,agentname=JMB_test2210,name=cpuPerUrl,parent=Welcome\ to\ Tomcat,path=tags value=0.0,units="%"
```

**Comment by beckettsean** [comment]
Also check your file to make sure you don't have any `0x0D` (carriage return) characters in your file. That is common when the file has passed through a Windows machine and will [cause issues](https://influxdb.com/docs/v0.9/troubleshooting/frequently_encountered_issues.html#getting-an-unexpected-error-when-sending-data-over-the-http-api) when submitting to InfluxDB.

**Comment by jmorantebriones** [comment]
Many thanks Sean!! After removing the quotes of tags it is working fine!! 

I was adding quotes because I thought that it was required as for the string values. I read again the documentation and noticed that it is not required to add quotes as the tags are always stored as strings.

FYI. I have confirmed that only the tags with escaped spaces are failing with the quotes. I suppose that something has changed since 0.9.0 as it was working in that version. 

I proceed to close the issue.

Thanks again!
